karyon3-api/src/main/java/com/netflix/karyon/spi/health/AbstractHealthIndicator.java

```diff
@@ -1,8 +1,11 @@
-package com.netflix.karyon.health;
+package com.netflix.karyon.spi.health;
 
 import java.util.Collections;
 import java.util.Map;
 
+import com.netflix.karyon.api.health.HealthIndicator;
```

Comment:
I'm having trouble reasoning about putting classes other than Interface (maybe some abstract classes) into the "api" package. Most abstract and all concrete classes should belong to the concrete impl package.